docs/change-log.md

```diff
@@ -2,11 +2,22 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update November 11th, 2022
+
+- **New:** Added autocompletion of [placeholders](https://docs.humanitec.com/reference/concepts/app-config/placeholders) in `labels`, `variables` and `files` properties of a workload in [Deployment Sets](sets-and-deltas.md#deployment-sets).
+
+- **Improved:** Performance of the `All deployment versions` functionality in the [App Details Screen](https://docs.humanitec.com/reference/user-interface/app-details-screen#overview) for applications with a large number of past deployments.
```

Comment:
I know we're not consistent with this but I guess we could start capitalising "Applications" as well